browser/src/slideshow/PauseTimer.ts

```diff
@@ -59,7 +58,7 @@ class PauseTimerGl extends Transition2d implements PauseTimer {
 		this.ctx = this.textCanvas.getContext('2d');
 
 		this.textTexture = this.createTextTexture(
-			Math.ceil(this.pauseTimeRemaining),
+			`Pause...( ${Math.ceil(this.pauseTimeRemaining)} )`,
```

Comment:
it was reverted?

Reply:
Yes, I think merge conflict override it.

Reply:
Thanks!